logging.md

```diff
@@ -0,0 +1,34 @@
+# Conventions for LOGGING
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All logging may be stored in files. 
+If so, they must be given in Provenance as a result. 
+
+## Log Levels
+A more detailed description will be found [here](https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels/5278006#5278006)
```

Comment:
Since this is actually pertinent to developers IMHO it should be added to https://ocr-d.github.io/cli#logging.

The `core` implementation doesn't distinguish between TRACE and DEBUG.

Reply:
True. Trace is only for completeness and should not be used in the workflow.